python/ray/air/config.py

```diff
@@ -522,8 +522,7 @@ class CheckpointConfig:
             on disk for this run. If a checkpoint is persisted to disk after
             there are already this many checkpoints, then an existing
             checkpoint will be deleted. If this is ``None`` then checkpoints
-            will not be deleted. If this is ``0`` then no checkpoints will be
-            persisted to disk.
+            will not be deleted. Must be a value >= 1.
```

Comment:
```suggestion
            will not be deleted. Must be >= 1.
```
Nit, but I think this flows better?